arviz/plots/posteriorplot.py

```diff
@@ -117,17 +117,25 @@ def plot_posterior(
     backend: str, optional
         Select plotting backend {"matplotlib","bokeh"}. Default "matplotlib".
     backend_kwargs: bool, optional
-        These are kwargs specific to the backend being used. For additional documentation
-        check the plotting method of the backend.
+        These are kwargs specific to the backend being used, passed to
+
+        :func:`matplotlib.pyplot.subplots` or
+        :func:`bokeh.plotting.figure`
     show: bool, optional
         Call backend show function.
     **kwargs
-        Passed as-is to plt.hist() or plt.plot() function depending on the value of `kind`.
+        Passed as-is to :func:`matplotlib.pyplot.hist` or :func:`matplotlib.pyplot.plot` function depending on the value of `kind`.
```

Comment:
That is great thanks! Can you also check the code in https://github.com/arviz-devs/arviz/blob/main/arviz/plots/backends/bokeh/posteriorplot.py to see where they are passed if the backend is bokeh and do the same?

Reply:
Um sorry but can you explain?

Reply:
Try following the steps in https://github.com/arviz-devs/arviz/wiki/ArviZ-Hacktoberfest-2021#adding-backend_kwargs, in https://github.com/arviz-devs/arviz/pull/1837#discussion_r721509870 or https://github.com/arviz-devs/arviz/pull/1837#discussion_r721528878 but for `kwargs` and in `plot_posterior` files.